source/_integrations/compensation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Compensation
+description: Instructions on how to integrate compensation sensors into Home Assistant.
+ha_category:
+  - Utility
+  - Sensor
+ha_iot_class: Local Push
+ha_release: '0.117'
+ha_quality_scale: internal
+ha_codeowners:
+  - '@petro31'
+ha_domain: compensation
+---
+
+The `compensation` sensor platform consumes the state from other sensors. It exports the `compensated` value as state and the following values as attributes: `entity_id` and `coefficients`.
+
+## Configuration
+
+To enable the compensation sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: compensation
+    name: Yamaha Zone 1 Volume
+    entity_id: media_player.yamaha_receiver
+    attribute: volume_level
+    unit_of_measurement: dB
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
fixed